### PR TITLE
Add Release Notes for Chocolatey GUI v2.1.1

### PR DESCRIPTION
**Pull request by corbob**
## Description Of Changes

This adds the release notes for Chocolatey GUI v2.1.1. It also updates the Chocolatey Component Dependencies chart with the latest release versions.

## Motivation and Context

To add release notes for the release.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [x] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left-hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/chocolatey-gui/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
...
+## 2.1.1 (April 29, 2024)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
+
+### Bug Fixes
+
+- Fix - Chocolatey GUI Not Displaying Error Output When Running as Non-Admin Through Background Service - see [#967](https://github.com/chocolatey/ChocolateyGUI/issues/967).
+- Fix - Chocolatey GUI Does Not Start if the Config Folder is Missing From C:\ProgramData\Chocolatey GUI. - see [#996](https://github.com/chocolatey/ChocolateyGUI/issues/996).
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:30]:
```suggestion
- Fix - Chocolatey GUI not displaying error output when running as non-admin through Background Service - see [#967](https://github.com/chocolatey/ChocolateyGUI/issues/967).
- Fix - Chocolatey GUI does not Start if the config folder is missing from `C:\ProgramData\Chocolatey GUI`. - see [#996](https://github.com/chocolatey/ChocolateyGUI/issues/996).
```

**Review by pauby — CHANGES_REQUESTED** [review]
We don't need to capitalise everything.

File: input/en-us/chocolatey-gui/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,16 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
...
+## 2.1.1 (April 29, 2024)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
+
+### Bug Fixes
+
+- Fix - Chocolatey GUI not displaying error output when running as non-admin through Background Service - see [#967](https://github.com/chocolatey/ChocolateyGUI/issues/967).
+- Fix - Chocolatey GUI does not Start if the config folder is missing from `C:\ProgramData\Chocolatey GUI`. - see [#996](https://github.com/chocolatey/ChocolateyGUI/issues/996).
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:30]:
```suggestion
- Fix - Chocolatey GUI does not start if the config folder is missing from `C:\ProgramData\Chocolatey GUI`. - see [#996](https://github.com/chocolatey/ChocolateyGUI/issues/996).
```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/chocolatey-gui/release-notes.md (hunk 2 above)
Comment (corbob) [reply input/en-us/chocolatey-gui/release-notes.md]: I've fixed this up and force pushed the amended commit.

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
Approved. Unsure of you want it merged now or not. Happy to do that if you let me know.

**Comment by corbob** [comment]
> Approved. Unsure if you want it merged now or not, now. Happy to do that if you let me know.

@pauby I'll work with Ops to get this merged once I get the release pushed to the Community Repository 👍

**Comment by Windos** [comment]
Good work, @corbob !

**Pull request merged**